examples/models/question_answering/Question_Answering.py

```diff
@@ -0,0 +1,572 @@
+
+scimago_jr = pd.read_csv('data/covid19data/scimagojr_2018.csv', sep=';')
+scimago_jr.drop_duplicates(['Title'], inplace=True)
+scimago_jr = scimago_jr.reset_index(drop=True)
+def QA(queries: dict, qa_model, num_papers = 5):
+        data = queries[0]
+        final_dict = {}
+        for task in data.keys():
+            print(f"Processing: {task}")
+            task_questions = data[task]['questions']
+            dict_task_quest = {}
+            dict_queries = {}
+            for idx, query in enumerate(task_questions):
+                print(f"Getting answers from query: {query}")
+                indexes_papers = extractPapersIndexes(query, num_papers=num_papers)
+                (question, indexes_papers) = (query, indexes_papers)
+                answers_list = []
+                for paper_index in indexes_papers:
+                    answer, conf, paragraph = answerQuestion(question, df_covid_only['body_text_parags'][
+                        paper_index], qa_model)  # best_answer, confidence, best_paragraph
+                    if answer:
+                        author = df_covid_only['authors'][paper_index] if not pd.isna(
+                            df_covid_only['authors'][paper_index]) else "not available"
+                        journal = df_covid_only['journal'][paper_index] if not pd.isna(
+                            df_covid_only['journal'][paper_index]) else "not available"
+                        title = df_covid_only['title'][paper_index] if not pd.isna(
+                            df_covid_only['title'][paper_index]) else "not available"
+                        start, end = findStartEndIndexSubstring(paragraph, answer)
+                        answer_parag = getSingleContext(paragraph, start, end)
+                        paper_citations_count, journal_api = getAPIInformations(df_covid_only['paper_id'][paper_index])
+                        journal = journal_api if journal_api else journal
+
+                        journal_row = scimago_jr[scimago_jr['Title'].apply(lambda x: x.lower()) == journal.lower()]
+                        journal_score = journal_row[
+                            'SJR'].item() if not journal_row.empty else "Information not available"
+                        paper_answer = {
+                            "answer": answer_parag,
+                            "title": title,
+                            "journal": journal,
+                            "author": author,
+                            "journal_score": journal_score,
+                            "paper_citations_count": paper_citations_count,
+                        }
+                        answers_list.append(paper_answer)
+                dict_queries[query] = answers_list
+            dict_task_quest['queries'] = dict_queries
+            dict_task_quest['task_name'] = data[task]['area']
+            final_dict[task] = dict_task_quest
+        full_code = final_dict
+        # return getHtmlCode(full_code, layoutStyle())
```

Comment:
why not call getHtmlCode()?
Can the front-end parse and display the returned table?

Reply:
Now it supports getHtmlCode(). Just by uncommenting this line while commenting out the following lines.
I optimised admin.py and dev.py files for singa-auto, to better run model efficiently & create_dataset() regarding to 'task == "question_answering", and made PR.